helper/logformat/vault.go

```diff
@@ -1,175 +1,45 @@
 package logformat
 
 import (
-	"encoding/json"
-	"fmt"
 	"io"
 	"os"
 	"strings"
-	"sync"
-	"time"
 
-	log "github.com/mgutz/logxi/v1"
+	log "github.com/hashicorp/go-hclog"
 )
 
-const (
-	styledefault = iota
-	stylejson
-)
+func NewVaultHCLogger(w io.Writer, level log.Level) log.Logger {
+	opts := &log.LoggerOptions{
+		Level:  level,
+		Output: w,
+	}
+	if useJson() {
+		opts.JSONFormat = true
```

Comment:
Maybe just add `JSONFormat: useJson()` to the initializer.

Reply:
Yes! Would read better that way, will do.